metabase/metabase
# 40895 entity path in palette

Closes https://github.com/metabase/metabase/issues/40895 

### Description
This PR adds collection and database names to search results and recent items in the command palette, as well as verified badges to items. This PR also adds keyboard navigation to the palette for `home`, `end`, `page up` and `page down` keys

### How to verify
1. Open the command palette and take a look at the recent items section, you should see the parent collections listed beside entities and database names listed beside tables.
2. Search for entities and tables, and you should see the same thing
3. Search for a verified question and you should see a verified badge next to the card name. 

### Demo
![chrome_nqyyj23XxM](https://github.com/metabase/metabase/assets/1328979/a1340864-a299-4bd0-b2b7-8e03866957de)

### Checklist
